### DISCUSSION TITLE
Questions about Stellarium

### DISCUSSION THREAD

**Discussion by matheissen** [Stellarium/stellarium#3475] [Q&A]
I have version 23.3.  It seems to me the sorting of prior entries in the search window is completely random.  It used to sort with the MRU at the top.  Now it's all over the place.
As a 2nd question - is it possible to make the equatorial grid display be J2000?  It seems to me that it's JNOW and I cant' figure out how to change it.

**Comment by alex-w** [comment]
> I have version 23.3. It seems to me the sorting of prior entries in the search window is completely random. It used to sort with the MRU at the top. Now it's all over the place.

The shortest items now upper

> As a 2nd question - is it possible to make the equatorial grid display be J2000? It seems to me that it's JNOW and I cant' figure out how to change it.

See F4->Markings or F2->Extras

**Reply by gzotti** [reply]
MRU would be more useful, I agree.

**Reply by 10110111** [reply]
Actually yes, this behavior does feel like broken. I did catch myself on the sensation that my last entry was not being registered as last. I used to bump them by simply choosing them, since this makes it easy to go to the last choice by <kbd>F3</kbd> → <kbd>Enter</kbd>.

**Reply by matheissen** [reply]
Yeah.  I’m hoping they change it back to MRU.  Sorting by the length of entry doesn’t do anything for me.

**Reply by alex-w** [reply]
Please check the latest weekly snapshot https://github.com/Stellarium/stellarium-data/releases/tag/weekly-snapshot
